standard/flags.lua

```diff
@@ -62,6 +62,54 @@ function Flags.Icon(args, flagName)
 	end
 end
 
+-- Returns the localisation/country-adjective of a country or region
+--[[
+supported args are:
+country			- country name, flag code, or alias of the Flag
+displayNoError		- boolean that decides if there should be a displayed error if no entry is found
+shouldReturnSimpleError	- boolean that decides if displayed error should be simple or detailed
+
+examples:
+Flags.getLocalisation({country = de}) = German
+Flags.getLocalisation({country = Germany}) = German
+Flags.getLocalisation({country = deu}) = German
+]]--
+function Flags.getLocalisation(args)
+	args = args or {}
+	local country = args.country
+
+	if String.isEmpty(country) then
+		return ''
+	end
+
+	local displayNoError = Logic.readBool(args.displayNoError)
+	local shouldReturnSimpleError = Logic.readBool(args.shouldReturnSimpleError)
```

Comment:
since (longterm) we want to replace https://liquipedia.net/commons/Module:Localisation with this PR i added the same options that existed there
simple error means it doesn't add the tracking category andf instead of the display it just returns `'error'`
there most likely are modules that check for the return value not being `'error'`